README.md

```diff
@@ -39,7 +39,7 @@ final class AppDelegate: ApplicationDelegate {
 **AppKid** depends on mulitple opensource projects. Below are the instructrions on how to set those up for Debian-based Linux distributions. RPM-Based instructions will be added some time later.
 
 - <details>
-	<summary>Swift language</summary>
+	<summary>Swift language [<--click here for instructions]</summary>
```

Comment:
I'm simply trying to find a suitable way to let people reading this for the first time, who are potentially new to this layout and/or swift to have awareness that important parts of the instructions are *not* visible on-screen in the readme when its first loaded. .,. because i'm really smart, been coding in 10+ languages for 30+ years, including web dev, and i missed it on my first read, so its not obvuous till youve seen it before